string colour
geometry_msgs/Pose pose
string direction
